synapse/rest/admin/users.py

```diff
@@ -195,8 +196,35 @@ def __init__(self, hs):
                 # add new threepids to user
                 current_time = self.hs.get_clock().time_msec()
                 for threepid in body["threepids"]:
+                    # For emails, transform the address to lowercase.
+                    # We store all email addreses as lowercase in the DB.
+                    # (See add_threepid in synapse/handlers/auth.py)
+                    address = threepid["address"].lower()
```

Comment:
Just noting that this seems related to #7021.

Reply:
Yes, it is.

Reply:
This change needs to wait on the result of that conversation then.